frontend/src/pages/profile-preferences-page/profile-preferences-page.tsx

```diff
@@ -0,0 +1,205 @@
+  const { firstName, lastName, username } = profile;
+
+  return (
+    <div className={style['preference-page-content-container']}>
+      {isNeedUpload && <UploadImage images={images} onUpload={onChange} onClose={onUploadImageClose} />}
+      {isNeedImageEditor && (
+        <ImageEditor
+          avatar={preparedAvatar}
+          setAvatar={setPreparedAvatar}
+          onClose={(): void => setIsNeedImageEditor(false)}
+          handleSave={onImageSave}
+        />
+      )}
+      <div className={style['profile-preferences-container']}>
+        <div className={style['profile-preferences-main-content-container']}>
+          <div className={style['image-setting-container']}>
+            <h2 className={style['profile-image-header']}>Profile image</h2>
+            <div className={style['profile-image-upload-container']}>
+              <div className={style['avatar-container']}>
+                {isLoading ? (
+                  <Loader spinnerSize={'xs'} vCentered={true} hCentered={true} />
+                ) : (
+                  <img
+                    className={style['profile-image-preview']}
+                    width={'96'}
+                    height={'96'}
+                    src={profile.avatar === '' ? defaultAvatar : profile.avatar}
+                    alt="profile picture preview"
+                  />
+                )}
+              </div>
+              <div className={style['image-upload-control-container']}>
+                <button
+                  type="button"
+                  className={style['image-upload-button']}
+                  onClick={(): void => {
+                    setIsNeedUpload(!isNeedUpload);
+                  }}
+                >
+                  Upload profile image
+                </button>
+                <span>
+                  {error ? (
+                    <span className={style['error-message']}>{error}</span>
+                  ) : (
+                    'Must be JPEG, PNG and cannot exceed 10MB.'
+                  )}
+                </span>
+              </div>
+            </div>
+          </div>
+          <div className={style['image-setting-container']}>
+            <h2 className={style['profile-image-header']}>Channel Banner</h2>
+            <div className={style['profile-image-upload-container']}>
+              <div className={style['channel-avatar-container']}>
+                <img
+                  className={style['profile-banner-preview']}
+                  height={'96'}
+                  src={defaultUserBanner}
+                  alt="profile picture preview"
+                />
+              </div>
+              <div className={style['image-upload-control-container']}>
+                <button type="button" className={style['image-upload-button']}>
+                  Update
+                </button>
+                <span>File format: JPEG, PNG (recommended 1200x480, max 10MB) </span>
+              </div>
+            </div>
+          </div>
```

Comment:
```suggestion
```
we don't have banner field for user, and channel banner will be set up on channel settings page instead of here.

Reply:
done